firmware/controllers/sensors/maf.h

```diff
@@ -2,7 +2,7 @@
  * @file    maf.h
  * @brief
  *
- * by the way 2.081989116 kg/h = 1 ft�/m
+ * by the way 2.081989116 kg/h = 1 ft�/m
```

Comment:
here we change risky but working symbol into something not working?

Reply:
ah geez this is the same thing as the degree symbols

Reply:
Hey look, there's even a bug filed for it on vscode!
https://github.com/Microsoft/vscode/issues/52642

Reply:
now we change risky but working symbol in to not-risky but working two symbols

Reply:
Could always abbreviate that to CFM, its a pretty common abbreviation in the aftermarket as its used for cylinder head flows all the time.